core/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java

```diff
@@ -109,4 +103,26 @@ public void testOsStats() {
             assertThat(stats.getSwap().getUsed().getBytes(), equalTo(0L));
         }
     }
+
+    public void testGetSystemLoadAverage() {
+        assumeTrue("test runs on Linux only", Constants.LINUX);
```

Comment:
You could probably avoid this by making the linux check a method that you stub out in OsProbe.